jujugui/static/gui/src/app/init/app.js

```diff
@@ -55,6 +55,8 @@ const UserMenu = require('../components/user-menu/user-menu');
 const USSOLoginLink = require('../components/usso-login-link/usso-login-link');
 const Zoom = require('../components/zoom/zoom');
 
+require('../assets/css/base.scss');
```

Comment:
I just loaded all our current css here, but really it would be imported per module. We could start like this however.